### PR TITLE
[symfony] add AutoWireWithClassNameSuffixForMethodWithRequiredAnnotationRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba what command to be run for auto-change all class in rector directory (src, packages,rules) to use this rule?

**Comment by TomasVotruba** [comment]
Like this: https://github.com/rectorphp/rector/blob/251789c82c04c325ef1468b8e563d67db6ce90bb/composer.json#L311

**Comment by samsonasik** [comment]
I got the following error in github action phpstan: 

```bash
Run composer phpstan
> vendor/bin/phpstan analyse --ansi --error-format symplify
Note: Using configuration file /home/runner/work/rector/rector/phpstan.neon.
Invalid configuration:
Unexpected option 'parameters › symplify'.
Script vendor/bin/phpstan analyse --ansi --error-format symplify handling the phpstan event returned with error code 1
```

seems related with latest symplify ?

**Comment by samsonasik** [comment]
@TomasVotruba could you verify the phpstan "Unexpected option 'parameters › symplify'."  error above? Thank you.

**Comment by TomasVotruba** [comment]
I'll update the `master` to fix the error

**Comment by samsonasik** [comment]
I think it is ready then

**Comment by TomasVotruba** [comment]
It should be ready in 3 minutes. I'll need you to rebase then

**Comment by TomasVotruba** [comment]
Ready to rebase

https://github.com/rectorphp/rector/pull/4342

**Comment by samsonasik** [comment]
rebased.

**Comment by samsonasik** [comment]
PHPUnit in PHP 7.2  error in master as well.

File: phpstan.neon [hunk 1]
```diff
@@ -630,47 +630,6 @@ parameters:
...
-                - src/Rector/AbstractRector/CallableNodeTraverserTrait.php # 24
-                - src/Rector/AbstractRector/ComplexRemovalTrait.php # 59
-                - src/Rector/AbstractRector/ComplexRemovalTrait.php # 59
-                - src/Rector/AbstractRector/ConstFetchAnalyzerTrait.php # 24
-                - src/Rector/AbstractRector/ConstFetchAnalyzerTrait.php # 24
-                - src/Rector/AbstractRector/NodeTypeResolverTrait.php # 59
-                - src/Rector/AbstractRector/NodeTypeResolverTrait.php # 59
-                - src/Rector/AbstractRector/VisibilityTrait.php # 29
-                - src/Rector/AbstractRector/VisibilityTrait.php # 29
-
```
Comment (TomasVotruba) [inline phpstan.neon:673]: :partying_face:

**Comment by TomasVotruba** [comment]
Thank you :+1:

File: rector-ci.php [hunk 2]
```diff
@@ -24,6 +24,7 @@
         SetList::DEFLUENT,
         SetList::TYPE_DECLARATION,
         SetList::PHPUNIT_CODE_QUALITY,
+        Setlist::SYMFONY_AUTOWIRE,
     ]);
```
Comment (TomasVotruba) [inline rector-ci.php:28]: :+1:

**Comment by TomasVotruba** [comment]
@dobryy Here it was added to Rector and code updated so `phpstan.neon` does not report it.

**Pull request merged**